src/Components/Aspire.Microsoft.Data.SqlClient/Aspire.Microsoft.Data.SqlClient.csproj

```diff
@@ -22,6 +22,7 @@
   <ItemGroup>
     <PackageReference Include="AspNetCore.HealthChecks.SqlServer" />
     <PackageReference Include="Microsoft.Data.SqlClient" />
+    <PackageReference Include="Microsoft.DotNet.RemoteExecutor" />
```

Comment:
@RussKie Assuming this was unintentional and likely a find/replace or similar issue? We need to undo this as it is obviously not okay to depend on this from the product code.

Reply:
Thank you for noticing; this doesn't look correct.